plugins/modules/wafv2_web_acl.py

```diff
@@ -341,8 +413,12 @@ def create(self, default_action, rules, sampled_requests, cloudwatch_metrics, me
                 'SampledRequestsEnabled': sampled_requests,
                 'CloudWatchMetricsEnabled': cloudwatch_metrics,
                 'MetricName': metric_name
-            }
+            },
+            'CustomResponseBodies': custom_response_bodies
         }
+
+        if len(custom_response_bodies) > 0:
```

Comment:
```suggestion
        if custom_response_bodies:
```

requires the changes below (remove default value)